pytest_blender/utils.py

```diff
@@ -60,15 +73,14 @@ def parse_version(version_string):
 
 
 def get_blender_version(blender_executable):
-    """Get Blender's version goving its executable location.
+    """Get the version of blender executable.
 
-    blender_executable : str
-      Blender's executable location.
+    Args:
+        blender_executable (str): Path to the blender executable file.
 
-    Returns
-    -------
+    Returns:
+        str: Blender's version in the format "3.1.2"
```

Comment:
The format is not guaranteed:

```suggestion
        str: Blender's version.
```